pkgs/games/prismlauncher/default.nix

```diff
@@ -46,51 +33,29 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-7tptHKWkbdxTn6VIPxXE1K3opKRiUW2zv9r6J05dcS8=";
   };
 
-  nativeBuildInputs = [ extra-cmake-modules cmake file jdk17 ninja wrapQtAppsHook ];
-  buildInputs = [
-    qtbase
-    qtsvg
-    zlib
-    quazip
-    ghc_filesystem
-    tomlplusplus
-  ]
-  ++ lib.optional (lib.versionAtLeast qtbase.version "6") qtwayland
-  ++ lib.optional gamemodeSupport gamemode.dev;
+  nativeBuildInputs = [ extra-cmake-modules cmake jdk17 ninja ];
+  buildInputs =
+    [
+      qtbase
+      zlib
+      quazip
+      ghc_filesystem
+      tomlplusplus
+    ]
+    ++ lib.optional gamemodeSupport gamemode;
```

Comment:
```suggestion
  buildInputs = [
    qtbase
    zlib
    quazip
    ghc_filesystem
    tomlplusplus
  ] ++ lib.optional gamemodeSupport gamemode;
```

Reply:
I wouldn't mind just reformatting everything with a strict formatter like `alejandra`. Are there any guidelines for this actually? Should I just leave a note somewhere for future maintainers about this?

Reply:
> I wouldn't mind just reformatting everything with a strict formatter like `alejandra`. Are there any guidelines for this actually? Should I just leave a note somewhere for future maintainers about this?

Often [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt) is used. I've not seen a guideline for this but I'd suggest it over alejandra because I've had PRs blocked while using alejandra for style issues

Reply:
Dropped my alejandra commit again.